packages/rocketchat-ui-message/client/message.coffee

```diff
@@ -120,7 +120,8 @@ Template.message.helpers
 			total = reaction.usernames.length
 			usernames = '@' + reaction.usernames.slice(0, 15).join(', @')
 
-			usernames = usernames.replace('@'+userUsername, t('You').toLowerCase())
+			reg = new RegExp('@'+userUsername+'$')
```

Comment:
You should escape the username cuz it could contains dots or other special characters.
`_.escapeRegExp()`